### PR TITLE
Lizards can get random verbs-the-noun names on character creation

### PR DESCRIPTION
**Pull request by Yenwodyah**
## About The Pull Request
So Grazyn [had the idea for this on the forums](https://tgstation13.org/phpBB/viewtopic.php?f=33&t=25295&start=50#p542273) and I thought it was neat so I replicated what they did and PR'ed it. Also I cleaned up the random name generating procs cause there was a bunch of copy-paste.

 - Lizards can get a random Verbs-The-Noun style name on character creation now
 - This has a 50% chance of happening, 50% of getting a classic-style random name
 - All the random unique name procs have been condensed into one under species, since they were all doing the same thing anyways.
 - I left the random unique human name proc as-is for convenience
 - Ashwalkers keep the old lizard names 100% of the time

## Why It's Good For The Game

 - I think most of the random names are better than what lizard players are coming up with right now
 - They're funny

Feel free to identify verbs/nouns you think should be removed/added, I didn't really look over the whole list.

## Changelog
:cl: Yenwodyah, Grazyn
add: Lizards can get random Verbs-The-Noun style names on character creation
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
I will probably close this after 24 hours.

I'm not a huge fan of this naming system over the argonian one, but I do accept it's allowed on the servers.

**Comment by RaveRadbury** [comment]
The execution of this idea is simple and elegant. Nice.

**Comment by Grazyn** [comment]
To clarify, the main idea was to have this system so that in the event of forced random names (like on manuel) lizards would still have access to traditional verb-the-noun names which have been part of server culture for many years now.

**Comment by Yenwodyah** [comment]
> I will probably close this after 24 hours.
> 
> I'm not a huge fan of this naming system over the argonian one, but I do accept it's allowed on the servers, but I'd rather not normalise it as the default randoms.

Fair enough ¯\_(ツ)_/¯

File: strings/names/lizard_verbs.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Strikes
+Orders
+Swallows
+Paralyzes
+Switches
+Tastes
+Pierces
+Tickles
+Places
+Pleases
```
Comment (LemonInTheDark) [inline strings/names/lizard_verbs.txt]: My mind is lodged firmly in the gutter

File: strings/names/lizard_nouns.txt [hunk 2]
```diff
@@ -0,0 +1,6796 @@
...
+Adaptation
+Adapter
+Addiction
+Addition
+Address
+Adjective
+Adjustment
+Admin
+Administration
+Administrator
```
Comment (LemonInTheDark) [inline strings/names/lizard_nouns.txt]: This will end well

File: strings/names/lizard_nouns.txt [hunk 3]
```diff
@@ -0,0 +1,6796 @@
...
+Admire
+Admission
```
Comment (LemonInTheDark) [inline strings/names/lizard_nouns.txt]: I am a moron but these feel wrong

File: strings/names/lizard_nouns.txt [hunk 4]
```diff
@@ -0,0 +1,6796 @@
...
+Function
+Functionality
+Fund
+Funding
+Fundraising
+Funeral
+Fur
+Furnace
+Furniture
+Furry
```
Comment (LemonInTheDark) [inline strings/names/lizard_nouns.txt]: Why

File: strings/names/lizard_verbs.txt [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Massages
+Melts
+Mocks
+Munches
+Murders
+Notices
+Judges
+Bends
+Owns
+Beats
```
Comment (LemonInTheDark) [inline strings/names/lizard_verbs.txt]: Just..... Just......

**Review by LemonInTheDark — COMMENTED** [review]
Saying "Read through the word list and pick out what stands out" does not seem to be a valid strat. You need to do a once over of these words, and remove the boomer moves/not nouns.

Idea seems ok, but these words are bull.

**Comment by XDTM** [comment]
Wait, there is another lizard naming convention besides verbs-the-noun?

**Comment by Couls** [comment]
I assume oranges means the tamrielic version of the elder scrolls argonian race naming
image

File: strings/names/lizard_verbs.txt (hunk 5 above)
Comment (Couls) [reply strings/names/lizard_verbs.txt]: Beats-The-Administrator

**Comment by tralezab** [comment]
would prefer a choice

**Comment by Tlaltecuhtli** [comment]
i think the ashlizards should get 100% the asdksk kasdj shhsc names while crew lizards should get the pleases-the-cowboy ones

**Comment by HomocidlePancakes** [comment]
Verbs-The-Noun is just an argonian name translated to english.

**Comment by Grazyn** [comment]
> Verbs-The-Noun is just an argonian name translated to english.

Oranges means more refined Argonian names which aren't just verbs-the-noun. To make them more argonian you would need something like verb-the/his/her/-adjective-noun, alternated with simple adjective-noun, verb-noun, and various mix thereof.

**Comment by evilcyborgmrt** [comment]
Oranges on forum "vampires aren't scifi enough I'm closing this thread nyaaa"

Oranges on github "this space lizard name generator isn't elder scrolls enough I'm closing this pr nyaaaaa"

File: strings/names/lizard_verbs.txt (hunk 5 above)
Comment (fludd12) [reply strings/names/lizard_verbs.txt]: At long last, I can have the anti-NT red-colored lizard with a trenchcoat and officer's hat.
Owns-The-Means

**Comment by Tharcoonvagh** [comment]
aw

**Comment by cacogen** [comment]
sucking my damn head

**Comment by Couls** [comment]
Kills-The-PullRequest 😭🤮🤮

**Comment by trollbreeder** [comment]
Banishes-The-Thot

**Pull request closed without merging**